docs/deploy.md

```diff
@@ -236,3 +236,30 @@ token=$(kubeadm token list | grep authentication,signing | awk '{print $1}')
 # join master on worker nodes
 kubeadm join --token $token ${master_ip}
 ```
+
+### Setting CNI network routes
+
+Containers across multi-node could be connected via direct route. You should setup the routes for all nodes, e.g. suppose one master node and two worker nodes:
```

Comment:
We are recommending run frakti bare metal, not on other IaaS, which causes significant performance penalty.